src/commands/new/contract.rs

```diff
@@ -65,8 +65,10 @@ mod tests {
 	#[test]
 	fn test_new_contract_command_execute_success() -> Result<()> {
 		let temp_contract_dir = tempfile::tempdir().expect("Could not create temp dir");
-		let command =
-			NewContractCommand { name: "test_contract".to_string(), path: Some(PathBuf::from(temp_contract_dir.path())) };
+		let command = NewContractCommand {
```

Comment:
All these formatting changes shouldnt be included in this PR, it adds noise to the review. I assume that they were added due to the CI change, but then they should be fixed in a separate PR.

Reply:
Yes, you are right. I run a `cargo fmt` in the last commit that added a lot of noise